homeassistant/components/light/xiaomi_miio.py

```diff
@@ -24,10 +24,16 @@
 DEFAULT_NAME = 'Xiaomi Philips Light'
 PLATFORM = 'xiaomi_miio'
 
+CONF_MODEL = 'model'
+
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
     vol.Required(CONF_TOKEN): vol.All(cv.string, vol.Length(min=32, max=32)),
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_MODEL, default=None): vol.In(
```

Comment:
Would it make sense to validate this just as a `cv.string`? Otherwise this list might grow pretty big?

Reply:
(and you're already validating it below)

Reply:
I prefer the list of allowed values. It will point to configuration issues early. The list want grow fast. At the moment there are 3 supported models out of 3 products on the market.